notebook/notebookapp.py

```diff
@@ -539,6 +540,20 @@ def start(self):
     _("Allow the notebook to be run from root user.")
 )
 
+flags['use_redirect_file']=(
+    {'NotebookApp' : {'use_redirect_file' : True}},
```

Comment:
Having a flag for this doesn't really make sense, because use redirect file is already the default, so `--use-redirect-file` wouldn't do anything.

You could make a flag to *disable* it, like `--no-use-redirect-file`, but I suspect it's the sort of thing you'd usually set in a config file anyway, so maybe it doesn't need a flag.

Reply:
Acknowledged and removed.